test/stream/api/source/update-001-ok/initial-etcd-kvs.txt

```diff
@@ -0,0 +1,78 @@
+<<<<<
+definition/sink/active/%%TEST_KBC_PROJECT_ID%%/%%TEST_DEFAULT_BRANCH_ID%%/my-source/my-sink
+-----
+{
+  "projectId": %%TEST_KBC_PROJECT_ID%%,
+  "branchId": %%TEST_DEFAULT_BRANCH_ID%%,
+  "sourceId": "my-source",
+  "sinkId": "my-sink",
+  "version": {
+    "number": 5,
+    "hash": "46739e92e00d7521",
+    "modifiedAt": "2006-01-02T15:04:05.123Z",
+    "description": "Rollback to version 2"
+  },
+  "type": "table",
+  "name": "Modified Name",
+  "description": "My Description",
+  "config": [
+    {
+      "key": "storage.level.local.volume.assignment.count",
+      "value": 1
+    },
+    {
+      "key": "storage.level.local.volume.assignment.preferredTypes",
+      "value": [
+        "default"
+      ]
+    },
+    {
+      "key": "storage.level.local.volume.sync.bytesTrigger",
+      "value": "100KB"
+    },
+    {
+      "key": "storage.level.local.volume.sync.checkInterval",
+      "value": "1ms"
+    },
+    {
+      "key": "storage.level.local.volume.sync.countTrigger",
+      "value": 100
+    },
+    {
+      "key": "storage.level.local.volume.sync.intervalTrigger",
+      "value": "100ms"
+    },
+    {
+      "key": "storage.level.local.volume.sync.mode",
+      "value": "disk"
+    },
+    {
+      "key": "storage.level.local.volume.sync.wait",
+      "value": false
+    }
+  ],
```

Comment:
This part can be removed (?) to simplify the test.

The `config` only overwrites default configuration for the specific sink.